content/post/event-triggered-customer-segmentation.md

```diff
@@ -0,0 +1,247 @@
++++
+title = "Event Triggered Customer Segmentation"
+date = 2018-07-17T10:40:34+02:00
+draft = false
+author = "rblasucci"
+description = "In which we quickly and easily create a full application to use Wallaroo to manage an ad campaign for a Marsupial Fan Club."
+tags = [
+    "Python",
+    "State",
+    "Wallaroo",
+]
+categories = [
+    "Python-Api",
+    "State",
+    "Stream-Processing",
+]
++++
+
+Today I'm going to show you how fast and easy it can be to set up a simple application with Wallaroo to manage an ad campaign.
```

Comment:
I think we should have a link from "Wallaroo" out to either the home page or one of the "Welcome to Wallaroo" blog posts in order to help orient folks.

I also think there should be something here that mentions that this post uses the Python API, with a link to the Python API section in the documentation.